### PR TITLE
update screenshots and steps

### PR DESCRIPTION
**Pull request by pb82**
Update UPS Firebase setup (site/aerogear/latest/push/google-setup.html). I also updated the screenshots because the old ones were no longer up to date with the Firebase UI.  One screenshot removed because there was no value in it.

### CODE REVIEW

File: modules/ROOT/pages/push/google-setup.adoc [hunk 1]
```diff
@@ -15,28 +15,26 @@ First, register with the Firebase Cloud Messaging (FCM) platform, formerly known
...
 +
 2. Now give your new project a name:
 +
-image:./img/project_name.png[New Project Name]
+image::project_name.png[New Project Name]
 +
 3. Once the project has been created you will be taken to the _Overview_ screen. From there click the Settings icon located next to the project name in the sidebar, and in the pop-up menu click **Project Settings**.
 +
-image:./img/project_settings.png[Project Number]
+image::img/project_settings.png[Project Settings]
```
Comment (josemigallas) [inline modules/ROOT/pages/push/google-setup.adoc]: Something wrong with this image

Comment (pb82) [reply modules/ROOT/pages/push/google-setup.adoc]: Fixed

File: modules/ROOT/pages/push/google-setup.adoc [hunk 2]
```diff
@@ -15,28 +15,26 @@ First, register with the Firebase Cloud Messaging (FCM) platform, formerly known
...
 +
-image:./img/retrieve_credentials.png[Retrieve Credentials]
+image::retrieve_credentials.png[Retrieve Credentials]
 +
-5. As the final step of the setup procedure, you need to download the `google-services.json` file. To do this, first navigate to the _Overview_ screen and click on **Add App** (If you have no apps connected to FCM yet, select the option to **Add Firebase to your Android app**).
+5. As the final step of the setup procedure, you need to download the `google-services.json` file. To do this, first navigate to the _Project Overview_ screen and click on **Add App** / **Add another App**. In the popup select _Add Firebase to your Android app_.
 +
-image:./img/add_app.png[Add Firebase to your app]
+6. In the pop-up window enter the Package name of your app and then click the **Register app** button. In the next step you can download the `google-services.json` file. This file needs to be built into your app binary. Download the JSON file and save it. To close the pop-up window, click **Next** on the following screens. Google will send an initial push message to your app for verification but you can skip this step if you haven't a running app yet.
```
Comment (josemigallas) [inline modules/ROOT/pages/push/google-setup.adoc]: Maybe it would be good to explicitly point out to ignore the "install Firebase SDK" step?

**Comment by josemigallas** [comment]
LGTM besides couple of minor comments.

**Comment by wtrocki** [comment]
@pb82 It's really sad that firebase do not provide documentation to this so it's really hard to avoid making many screenshots that could outdate. Really documentation. Please let me know if you will need help with IOS one.

**Comment by pb82** [comment]
@finp @jstaffor  good to merge?

**Comment by wtrocki** [comment]
After extensive searching i found article that does that instruction:
https://support.google.com/firebase/answer/7015592?hl=en

Maybe it will be useful. We simplified IOS by pointing to the external links.

**Pull request merged**